elide-core/src/main/java/com/yahoo/elide/core/RequestScope.java

```diff
@@ -335,6 +336,22 @@ protected RequestScope(String path, JsonApiDocument jsonApiDocument, RequestScop
     }
 
     /**
+     * Merge the global filterExpression with the one provided.
+     * @param filterExpression the filterExpression to merge
+     * @return The global filter expression evaluated at the first load
+     */
+    public Optional<FilterExpression> addGlobalFilterExpression(FilterExpression filterExpression) {
```

Comment:
Check out getLoadFilterExpression in RequestScope.  This is the method that probably should be augmented to AND the globalFilterExpression with the filter expression from the active permission executor.